concepts/routines-code-blocks-signatures/introduction.md

```diff
@@ -56,6 +56,19 @@ multi foo () { say 'No arguments given'}
 multi foo ($bar) { say "One argument given: $bar" }
 ```
 
+## WhateverCode
+
+When a `Whatever` star (`*`) is used in combination with most operators, a `WhateverCode` object is created.
+This allows you to create blocks with a simpler syntax.
+
+```raku
+# These will all increment an argument by 1:
```

Comment:
Can we bring attention to the 4th example somehow?  I knew what I was looking for, and visually, it was hard to spot.

Reply:
Nice.